route-rs-runtime/src/link/primitive/black_hole_link.rs

```diff
@@ -0,0 +1,91 @@
+use crate::link::{Link, LinkBuilder, PacketStream};
```

Comment:
I thought we got rid of the black hole links in favor of moving that logic to Processors?

Reply:
There's no way to poll the links, so we'd have to do something really strange like join the drop path back into one of the actual flows of the router.